armory/scenarios/base.py

```diff
@@ -122,8 +123,9 @@ def _send_to_mongo(self, mongo_host: str, output: dict):
         client = pymongo.MongoClient(mongo_host, MONGO_PORT)
         db = client[MONGO_DATABASE]
         col = db[MONGO_COLLECTION]
+        mongo_ip = re.findall(r"@([^@]*$)", mongo_host)[0]
```

Comment:
if `mongo_ip` does not have an `@` (eg an unauthed server), will `findall()` return a list with at least one element?

Reply:
if the regex doesn't find any `@`s, the `mongo_ip` can be printed as-is, since that would mean that there's no username or password included